.travis.yml

```diff
@@ -31,7 +31,8 @@ after_failure:
 - tests/scripts/travis-log-failure.sh
 env:
   global:
-    secure: "barHldniAfXyoWOD/vcO+E6/Xm4fmcaUoC9BeKW+LwsHqlDMLvugaJnmLXkSpkbYhVL61Hzf3bo0KPJn88AFc5Rkf8oYHPjH4adMnVXkf3B9ghHCgznqHsAH3choo6tnPxaFgOwOYmLGb382nQxfE5lUdvnM/W/psQjWt66A1+k="
+    - SEED=1
```

Comment:
Don't we want seed 4 these days?

Reply:
Nope it's back to 1 since we fixed the issue with tests that triggered bugs in older GnuTLS versions: in [all.sh](https://github.com/ARMmbed/mbedtls/blob/development/tests/scripts/all.sh#L343) and the [Jenkins job](https://github.com/ARMmbed/mbedtls-test/blob/master/vars/mbedtls.groovy#L35)